src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/weapons/SeismicAxe.java

```diff
@@ -150,7 +151,7 @@ private void pushEntity(Player p, Entity entity) {
 
     private @Nonnull Block findGround(@Nonnull Block b) {
         if (b.getType() == Material.AIR) {
-            for (int y = 0; y < b.getY(); y++) {
+            for (int y = 0; b.getY() - y > WorldUtils.getMinHeight(b.getWorld()); y++) {
```

Comment:
A small thing, but you could probably create variables for these calls to improve performance a little bit

Reply:
This one, yep, a variable makes sense.